sync/models/sync_project.py

```diff
@@ -80,6 +80,7 @@ def _compute_eval_context_description(self):
         for r in self:
             if not r.eval_context:
                 r.eval_context_description = ""
+                continue
```

Comment:
:+1: 
It was fixed in later versions https://github.com/itpp-labs/sync-addons/commit/6df763ddb4c3b74b1c448733838db2960d61d75a